Sonos/module.php

```diff
@@ -116,13 +166,18 @@ public function RequestAction($Ident, $Value)
 		//Remove on next Symcon update
 		protected function RegisterProfileInteger($Name, $Icon, $Prefix, $Suffix, $MinValue, $MaxValue, $StepSize) {
 		
-			if(!IPS_VariableProfileExists($Name)) {
-				IPS_CreateVariableProfile($Name, 1);
-			} else {
-				$profile = IPS_GetVariableProfile($Name);
-				if($profile['ProfileType'] != 1)
-					throw new Exception("Variable profile type does not match for profile ".$Name);
-			}
+//			if(!IPS_VariableProfileExists($Name)) {
+//				IPS_CreateVariableProfile($Name, 1);
+//			} else {
+//				$profile = IPS_GetVariableProfile($Name);
+//				if($profile['ProfileType'] != 1)
+//					throw new Exception("Variable profile type does not match for profile ".$Name);
+//			}
+
```

Comment:
Das geht leider nicht, da diese Funktion bald in der IPSModule Klasse sein wird und ich diese gerne hier entfernen würde. Ich würde das Löschen dieses einen Profils explizit vor dem erneuten erstellen machen und diese Funktion unberührt lassen.